extensions/amp-story/1.0/amp-story.js

```diff
@@ -1322,6 +1333,38 @@ export class AmpStory extends AMP.BaseElement {
     this.activePage_.setState(pageState);
   }
 
+  /**
+   * Reacts to sidebar state updates.
+   * @param {boolean} sidebarState
+   * @private
+   */
+  onSidebarStateUpdate_(sidebarState) {
+    const actions = Services.actionServiceForDoc(this.getAmpDoc());
+    if (this.win.MutationObserver) {
+      if (!this.sidebarObserver_) {
+        this.sidebarObserver_ = new this.win.MutationObserver(mutationsList => {
+          mutationsList.forEach(function(mutation) {
+            if (mutation.attributeName == 'open') {
+              this.storeService_.dispatch(Action.TOGGLE_SIDEBAR,
+                  this.sidebar_.hasAttribute('open'));
+              return;
+            }
+          }.bind(this));
+        });
+      }
+      if (this.sidebar_ && sidebarState) {
+        this.sidebarObserver_.observe(this.sidebar_, {attributes: true});
+        actions.execute(this.sidebar_, 'open', /* args */ null,
+        /* source */ null, /* caller */ null, /* event */ null,
+            ActionTrust.HIGH);
+      } else {
+        this.sidebarObserver_.disconnect();
+      }
+    } else if (sidebarState) {
+      this.storeService_.dispatch(Action.TOGGLE_PAUSED, false);
```

Comment:
Now that the `PAUSED_STATE` is toggled by the `TOGGLE_SIDEBAR` action, we should replace the dispatched action here.
The issue would be that, for a browser that doesn't support `MutationObserver`, the story would be unpaused, but the `SIDEBAR_STATE` still set to `true`, making it impossible for the user to re open the sidebar later.